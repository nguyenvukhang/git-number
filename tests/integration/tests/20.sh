# complex short status

init 100
_gitnu status -s
_gitnu add 25-75 # effectively adds files 25-50 inclusive
_gitnu status -s
_gitnu reset 10-20 # resets files 34-44 inclusive
log gitnu status -s
