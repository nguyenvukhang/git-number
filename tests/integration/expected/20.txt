1  [32mA[m  file_0025
2  [32mA[m  file_0026
3  [32mA[m  file_0027
4  [32mA[m  file_0028
5  [32mA[m  file_0029
6  [32mA[m  file_0030
7  [32mA[m  file_0031
8  [32mA[m  file_0032
9  [32mA[m  file_0033
10 [32mA[m  file_0045
11 [32mA[m  file_0046
12 [32mA[m  file_0047
13 [32mA[m  file_0048
14 [32mA[m  file_0049
15 [32mA[m  file_0050
16 [31m??[m file_0001
17 [31m??[m file_0002
18 [31m??[m file_0003
19 [31m??[m file_0004
20 [31m??[m file_0005
21 [31m??[m file_0006
22 [31m??[m file_0007
23 [31m??[m file_0008
24 [31m??[m file_0009
25 [31m??[m file_0010
26 [31m??[m file_0011
27 [31m??[m file_0012
28 [31m??[m file_0013
29 [31m??[m file_0014
30 [31m??[m file_0015
31 [31m??[m file_0016
32 [31m??[m file_0017
33 [31m??[m file_0018
34 [31m??[m file_0019
35 [31m??[m file_0020
36 [31m??[m file_0021
37 [31m??[m file_0022
38 [31m??[m file_0023
39 [31m??[m file_0024
40 [31m??[m file_0034
41 [31m??[m file_0035
42 [31m??[m file_0036
43 [31m??[m file_0037
44 [31m??[m file_0038
45 [31m??[m file_0039
46 [31m??[m file_0040
47 [31m??[m file_0041
48 [31m??[m file_0042
49 [31m??[m file_0043
50 [31m??[m file_0044
... 50 hidden items (gitnu)
